app/assets/javascripts/controllers/container_dashboard/util/chart-config-util.js

```diff
@@ -0,0 +1,22 @@
+var chartConfig = {
```

Comment:
Uh, can we refactor this please? We really don't want to introduce new globals.. maybe use `angular.module('containerDashboard').constant('chartConfig', { ... })` (and inject when needed)?

The other option would be to make it explicit by putting it under the ManageIQ global object, but I'm guessing `.constant` (or `.value`) works, right?